app/javascript/dashboard/components/app/PaymentPendingBanner.vue

```diff
@@ -34,6 +34,9 @@ export default {
     actionButtonMessage() {
       return this.$t('GENERAL_SETTINGS.OPEN_BILLING');
     },
+    isOnboardingView() {
```

Comment:
This shouldn't be handled here, it should be at the parent component. You can see the logic being repeated in two places.

Reply:
@pranavrajs The parent doesn't handle any v-if conditions and all the v-if conditions are written here. If we move just this condition it would create inconsistencies IMHO.